src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -179,6 +180,9 @@ public static class ScriptConstants
 
         public const string KubernetesManagedAppName = "K8SE_APP_NAME";
         public const string KubernetesManagedAppNamespace = "K8SE_APP_NAMESPACE";
+        public const string DiagnosticEventKey = "MS_DiagnosticEvent";
+        public const string HelpLinkKey = "helpLink";
+        public const string ErrorCodeKey = "errorCode";
```

Comment:
Can we make these also start with `MS_` to avoid collisions?

Reply:
done